tests/dataaccess/common/settings.q

```diff
@@ -0,0 +1,12 @@
+testpath:hsym`$getenv[`KDBTESTS],"/dataaccess/common";
+processcsv:` sv testpath,`config`process.csv;
+
+//- code to pass in a test name
+//- extract the input parameter from {testname}.csv
+//- extract the expected error from checkinputerrors.csv
+//- compare error with expected error
+checkreturnederror:{[test]errors[test;`error]~@[.checkinputs.checkinputs;gettestparams test;::]};
```

Comment:
Is this not common unit test code?

Reply:
No our errors are only caught in one place- checkinputs, hence we only need to trap errors in this script. It is not part of the [greater framework](https://github.com/AquaQAnalytics/TorQ/blob/dataaccessapi/tests/helperfunctions.q) or [K4](https://code.kx.com/q/kb/unit-tests/)